test/index.js

```diff
@@ -65,6 +65,17 @@ test('logos path names should not contain space', function (t) {
   t.end()
 })
 
+test('symbols should be five or less characters', function (t) {
+  Object.keys(contractMap).forEach(address => {
+    const contract = contractMap[address]
+    const symbol = contract.symbol
+    if (symbol) {
+      t.ok(symbol.length < 6, `symbol with more than 5 characters: "${symbol}"`)
```

Comment:
I'd test for the negative case > 5, so you only print what's wrong.

Reply:
Oh, nice catch! Didn't see that, I'll update it. Thanks!